### DISCUSSION TITLE
Folds

### DISCUSSION THREAD

**Discussion by jghauser** [nvim-neorg/neorg#420] [Q&A] [answered]
I noticed that Neorg only folds `document.meta` now and saw a commit that said this is now by "default" so. Is it possible to configure Neorg to fold other elements too? Or is folding currently under construction? :)

I do appreciate work on folding, it's been quite unreliable sometimes (especially with carryover tags i think). I've also never really been a fan of folding for lists.

**Comment by max397574** [comment, marked as answer]
folding should work with treesitter like this:
```lua
opt.foldmethod = "expr" -- use treesitter for folding
opt.foldexpr = "nvim_treesitter#foldexpr()"
```

**Reply by vhyrro** [reply]
That's not what @jghauser means tho, neorg autoenables folding for you :)

Folds are only configurable thru `after/queries/norg/folds.scm` in your config, I'll be improving them somewhere after 0.1 cause they've been bugging me a lot recently 😅. I say after 0.1 cause we're decently close now and I just want it to release already :p

**Reply by jghauser** [reply]
Yes, folding is already enabled and working, just not for all the elements I would like. Thanks for the clarification! I think for the time being I'll just have a custom folds.scm as you suggested.
